### PR TITLE
Fixed Typographical Error in guide/english/java/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by iamrajiv** [comment]
@RandellDawson can you review this

**Comment by iamrajiv** [comment]
@thecodingaviator can you review this

File: guide/english/java/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: Java
...
 
 ## OBJECT-ORIENTED PROGRAMMING LANGUAGE
-JAVA is one of the OBJECT ORIENTED PROGRAMMING LANGUAGES, unlike C and FORTAN which are procedural languages. The OBJECT ORIENTED concept of programming makes it much more efficient than procedural languages. Basically, object orientation breaks the program into parts, which is decided by the programmer. Then the complete program is stored in the permanent memory and when it is executed only that part is brought to dynamic memory which is called by the programmer. Each such part is called a `class` and when it is called in the dynamic memory it is called an `object`.
+JAVA is one of the OBJECT-ORIENTED PROGRAMMING LANGUAGES, unlike C and FORTRAN which are procedural languages. The OBJECT ORIENTED concept of programming makes it much more efficient than procedural languages. Basically, object orientation breaks the program into parts, which is decided by the programmer. Then the complete program is stored in the permanent memory and when it is executed only that part is brought to dynamic memory which is called by the programmer. Each such part is called a `class` and when it is called in the dynamic memory it is called an `object`.
```
Comment (Manish-Giri) [inline guide/english/java/index.md]:
```suggestion
JAVA is one of the object-oriented programming languages, unlike C and FORTRAN which are procedural languages. The object-oriented concept of programming makes it much more efficient than procedural languages. Basically, object orientation breaks the program into parts, which is decided by the programmer. Then the complete program is stored in the permanent memory, and when it is executed, only that part is brought to dynamic memory which is called by the programmer. Each such part is called a `class` and when it is called in the dynamic memory, it is called an `object`.
```

File: guide/english/java/index.md [hunk 2]
```diff
@@ -85,9 +85,9 @@ Verify Java has been properly installed in your system by opening Command Prompt
...
+After finishing the installation, we can begin to understand the nitty gritty of the Java ecosystem. Java is an <a href='http://stackoverflow.com/questions/1326071/is-java-a-compiled-or-an-interpreted-programming-language' target='_blank' rel='nofollow'>interpreted and compiled</a> language, that is the code we write gets compiled to bytecode and interpreted to run. We write the code in .java files, Java compiles them into <a href='https://en.wikipedia.org/wiki/Java_bytecode' target='_blank' rel='nofollow'>bytecodes</a> which are run on a Java Virtual Machine or JVM for execution. These bytecodes typically have a .class extension.
```
Comment (Manish-Giri) [inline guide/english/java/index.md]:
```suggestion
After finishing the installation, we can begin to understand the nitty gritty of the Java ecosystem. Java is an <a href='http://stackoverflow.com/questions/1326071/is-java-a-compiled-or-an-interpreted-programming-language' target='_blank' rel='nofollow'>interpreted and compiled</a> language, that is, the code we write gets compiled to bytecode and interpreted to run. We write the code in `.java` files, Java compiles them into <a href='https://en.wikipedia.org/wiki/Java_bytecode' target='_blank' rel='nofollow'>bytecode</a> which is run on a Java Virtual Machine or JVM for execution. The bytecode typically has a `.class` extension.
```

File: guide/english/java/index.md [hunk 3]
```diff
@@ -85,9 +85,9 @@ Verify Java has been properly installed in your system by opening Command Prompt
...
 
-Java is a pretty secure language as it doesn't let your program run directly on the machine. Instead, your program runs on a Virtual Machine called JVM. This Virtual Machine exposes several APIs for low level machine interactions you can make, but other than that you cannot play with machine instructions explicitely. This adds a huge bonus of security.
+Java is a pretty secure language as it doesn't let your program run directly on the machine. Instead, your program runs on a Virtual Machine called JVM. This Virtual Machine exposes several APIs for low-level machine interactions you can make, but other than that you cannot play with machine instructions explicitly. This adds a huge bonus of security.
```
Comment (Manish-Giri) [inline guide/english/java/index.md]:
```suggestion
Java is a pretty secure language as it doesn't let your program run directly on the machine. Instead, your program runs on a virtual machine called the JVM (Java Virtual Machine). This virtual machine exposes several APIs for low-level machine interactions you can make, but other than that, you cannot play with machine instructions explicitly. This adds a huge bonus of security.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
@iamrajiv Please address the requested changes.

**Comment by iamrajiv** [comment]
@Manish-Giri can you take a look, please

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**